src/fragments/lib-v1/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -158,6 +158,12 @@ import flutter7 from '/src/fragments/lib-v1/graphqlapi/flutter/advanced-workflow
 
 By default, the API plugin includes appropriate authorization headers on your outgoing requests. However, you may have an advanced use case where you wish to send additional request headers to AppSync.
 
+<Callout>
+
+If your API does not require any authorization or if you would like manipulate the request yourself, please refer to the [Set authorization mode to NONE](/gen1/[platform]/build-a-backend/graphqlapi/customize-authz-modes/#none)
```

Comment:
The link is to vCurrent.

I don't think we need to port this over from vCurrent to vPrevious. Let's revert changes to this file.